packages/pillarbox-playlist/src/pillarbox-playlist.js

```diff
@@ -78,13 +97,20 @@ class PillarboxPlaylist extends Plugin {
    */
   constructor(player, options) {
     super(player);
+
     if (options.playlist && options.playlist.length) {
       player.ready(() => {
         this.load(...options.playlist);
       });
     }
+
     this.autoadvance = !!options.autoadvance;
     this.repeat = !!options.repeat;
```

Comment:
with `options` set to `{}` we could do `Boolean(options.autoadvance)`

Reply:
As per my previous comment, #25 already takes care of this.